add ga iam role for s3 write (#2)
CrowdStrike/aws-security-lake: README.md

In this step, you'll run a script that will register CrowdStrike sources for each supported OCSF Event Class with Amazon Security Lake.

> This script will also create a IAM role that will be used later to write data to your Amazon Security Lake bucket.

Reviewer: ```suggestion
This script will also create an IAM role that will be used to write data to your Amazon Security Lake bucket.
```
